content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -412,8 +412,7 @@ same name.
 
```

Comment:
Please update the above paragraph as well.  "Those parameters are accessible as members of the `params` variable."